batch/test/test_dag.py

```diff
@@ -370,3 +371,24 @@ def test_input_dependency(client):
                             parent_ids=[head.id])
     tail.wait()
     assert tail.log()['main'] == 'head1\nhead2\n'
+
+
+def test_authorized_users_only():
+    endpoints = [
+        (requests.post, '/jobs/create'),
+        (requests.get, '/jobs'),
+        (requests.get, '/jobs/0'),
+        (requests.get, '/jobs/0/log'),
+        (requests.delete, '/jobs/0/delete'),
+        (requests.post, '/jobs/0/cancel'),
+        (requests.post, '/batches/create'),
+        (requests.get, '/batches/0'),
+        (requests.delete, '/batches/0/delete'),
+        (requests.post, '/batches/0/close'),
+        (requests.post, '/pod_changed'),
+        (requests.post, '/refresh_k8s_state'),
+        (requests.get, '/recent')]
+    for endpoint in endpoints:
```

Comment:
I prefer `for f, path in endpoints: ...`.

Reply:
done, also moved to test_batch since it's not a dag test.